ledger/lruaccts.go

```diff
@@ -80,6 +80,9 @@ func (m *lruAccounts) readNotFound(addr basics.Address) bool {
 // thread locking semantics : write lock
 func (m *lruAccounts) flushPendingWrites() {
 	pendingEntriesCount := len(m.pendingAccounts)
+	if pendingEntriesCount == 0 {
+		return
+	}
```

Comment:
WLOG take this as example: if `pendingEntriesCount` is length 0, nothing to flush in, fast exit; when the LRU cache is disabled, `m.pendingAccounts` can only be length 0, so we exit early before warning.